source/modulo_controllers/include/modulo_controllers/BaseControllerInterface.hpp

```diff
@@ -532,8 +579,13 @@ class BaseControllerInterface : public controller_interface::ControllerInterface
   std::map<std::string, std::function<void(const std::string&, const std::string&)>>
       custom_input_configuration_callables_;
 
-  std::shared_ptr<tf2_ros::Buffer> tf_buffer_;             ///< TF buffer
-  std::shared_ptr<tf2_ros::TransformListener> tf_listener_;///< TF listener
+  std::shared_ptr<tf2_ros::Buffer> tf_buffer_;                                ///< TF buffer
+  std::shared_ptr<tf2_ros::TransformListener> tf_listener_;                   ///< TF listener
+  std::shared_ptr<tf2_ros::TransformBroadcaster> tf_broadcaster_;             ///< TF broadcaster
+  std::shared_ptr<tf2_ros::StaticTransformBroadcaster> static_tf_broadcaster_;///< TF static broadcaster
+
+  std::shared_ptr<rclcpp::node_interfaces::NodeParametersInterface> node_parameters_;
```

Comment:
I'd rather we dont save these interfaces explicitly. This is only in the component interface because we dont have a real node yet